fabric_bolt/projects/views.py

```diff
@@ -372,10 +372,15 @@ def get_form(self, form_class):
 
             str_config_key = 'configuration_value_for_{}'.format(name)
 
-            field = CharField(label='Argument value for ' + name, initial=default)
+            if not config.prompt_me_for_input:
```

Comment:
The config variable is out of scope in this context. So even if the code "works" it certainly can't be expected to behave in a predictable way. So we can't accept this PR as is. I'm also not seeing this bug in the latest version. So you might want to make sure you're on the latest version.